src/App.vue

```diff
@@ -206,6 +223,13 @@ export default defineComponent({
           this.$store.dispatch('zap/updateSelectedUcComponentState', resp)
         }
       )
+
+      this.$onWebSocket(
+        dbEnum.wsCategory.dirtyFlag,
```

Comment:
The dirty flag is constantly polled from the backend already via web socket

Reply:
Yes, but there's no reflection of that in the UI state, so we can't conditionally show HTML UI elements based on that.
(These changes are done in order to embed ZAP into Studio v6, where there's no electron menu and potentially multiple ZAP UI instances in the same browser instance (sharing the session id)...)